### ISSUE TITLE
Logging into the cluster: what is the actual key file?

### ISSUE THREAD

**Issue by jowens** [amplab/training#75]
`ssh -i <key_file> root@<master_node_hostname>`

Is that the .pem file? Perhaps you could say so. (What are the .pem and .ppk files you sent us?)

**Comment by shivaram** [comment]
Yes, the key_file is the .pem file on Linux/OS X. For Putty on windows you will need to use the .ppk files. We will clarify this in the text though. Thanks for reporting
